aws/resource_aws_kinesis_analyticsv2_application.go

```diff
@@ -0,0 +1,2177 @@
+																Type:     schema.TypeString,
+																Optional: true,
+															},
+
+															"name": {
+																Type:     schema.TypeString,
+																Required: true,
+															},
+
+															"sql_type": {
+																Type:     schema.TypeString,
+																Required: true,
+															},
+														},
+													},
+												},
+
+												"record_encoding": {
+													Type:     schema.TypeString,
+													Optional: true,
+												},
+
+												"record_format": {
+													Type:     schema.TypeList,
+													Required: true,
+													MaxItems: 1,
+													Elem: &schema.Resource{
+														Schema: map[string]*schema.Schema{
+															"mapping_parameters": {
+																Type:     schema.TypeList,
+																Optional: true,
+																MaxItems: 1,
+																Elem: &schema.Resource{
+																	Schema: map[string]*schema.Schema{
+																		"csv": {
+																			Type:     schema.TypeList,
+																			Optional: true,
+																			MaxItems: 1,
+																			Elem: &schema.Resource{
+																				Schema: map[string]*schema.Schema{
+																					"record_column_delimiter": {
+																						Type:     schema.TypeString,
+																						Required: true,
+																					},
+
+																					"record_row_delimiter": {
+																						Type:     schema.TypeString,
+																						Required: true,
+																					},
+																				},
+																			},
+																		},
+
+																		"json": {
+																			Type:     schema.TypeList,
+																			Optional: true,
+																			MaxItems: 1,
+																			Elem: &schema.Resource{
+																				Schema: map[string]*schema.Schema{
+																					"record_row_path": {
+																						Type:     schema.TypeString,
+																						Required: true,
+																					},
+																				},
+																			},
+																		},
+																	},
+																},
+															},
+
+															"record_format_type": {
+																Type:     schema.TypeString,
+																Computed: true,
+															},
+														},
+													},
+												},
+											},
+										},
+									},
+
+									"starting_position_configuration": {
+										Type:     schema.TypeList,
+										Computed: true,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"starting_position": {
+													Type:     schema.TypeString,
+													Computed: true,
+												},
+											},
+										},
+									},
+
+									"stream_names": {
+										Type:     schema.TypeSet,
+										Computed: true,
+										Elem:     &schema.Schema{Type: schema.TypeString},
+									},
+								},
+							},
+						},
+
+						"outputs": {
```

Comment:
Given that there can be more than one of these and order doesn't seem to be important, `Type: schema.TypeSet` seems more appropriate.

Also, rename to `output`.